easybuild/easyconfigs/m/MariaDB-connector-c/MariaDB-connector-c-2.3.4-iomkl-2018a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'MariaDB-connector-c'
+version = '2.3.4'
```

Comment:
@edmondac Bump to `2.3.5`?

Why are you including this easyconfig here though, this is not a dependency for Bioconductor?